clients/privacy-center/components/Card.tsx

```diff
@@ -57,13 +57,7 @@ const Card: React.FC<CardProps> = ({
     >
       {title}
     </Text>
-    <Text
-      color="gray.600"
-      fontSize="xs"
-      fontWeight="normal"
-      lineHeight="16px"
-      noOfLines={3}
-    >
+    <Text color="gray.600" fontSize="xs" fontWeight="normal" lineHeight="16px">
```

Comment:
the PR description says the number of lines was set to 5, but I'm not seeing that here? seems like this one would be the unlimited size one?

Reply:
if it were up to me, I'd probably go with the unlimited size. and then if a user doesn't like the way it looks, it can be a forcing function to make their description more concise :) 

another option would be to not have the others grow, but then you'll likely have cards of different heights even when there's only a few lines of difference, which wouldn't be very aesthetic